recipes/sicer2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "SICER2" %}
+{% set version = "1.0.1" %}
+{% set pypi_hash = "5d/ea/4da9fde475d3a4b236f0c359f275c369333799d3519de9ce0b0fbaf5714c" %}
+{% set github = "https://github.com/zanglab/SICER2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/{{ pypi_hash }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b85dcac7c0711a89b699620e424734c58523d0bd42e49f4f5b7aeb822ec4d1dc
+
+build:
+  number: 0
+  # noarch: python
+  script: "{{ PYTHON }} -m pip install --ignore-installed --no-deps ."
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python >=3.6
+    - scipy
+    - setuptools
+  run:
+    - numpy
+    - python
+    - scipy
+
+test:
+  imports:
+    - sicer
+
+about:
+  home: "https://pypi.org/project/SICER2/"
+  dev_url: "{{ github }}"
```

Comment:
Why do you introduce the extra variable here? IMO, that only makes the recipe longer more complicated than necessary.

Reply:
I first drafted the recipe to build off the GitHub release (https://github.com/zanglab/SICER2/releases) instead of the pip release -- that's why the variable was there in the recipe